crates/driver/src/tests/cases/solve.rs

```diff
@@ -13,9 +13,10 @@ use {
 #[ignore]
 #[tokio::test]
 async fn test() {
+    crate::boundary::initialize_tracing("driver=trace");
     // Set up the uniswap swap.
     let setup::blockchain::Uniswap {
-        web3,
+        web3: _web3,
```

Comment:
If you don't need it, you can just get rid of it and add `..` at the end of the pattern.